app/form_models/managed_organisations_form.rb

```diff
@@ -2,13 +2,14 @@ class ManagedOrganisationsForm
   include ActiveModel::Model
   include OrganisationHelper
 
-  attr_accessor :managed_organisations, :managed_school_ids
+  attr_accessor :managed_organisations, :managed_school_ids, :managed_school_names
```

Comment:
there needs to be one array `:managed_schools=>[id => "0345c1a5-0522-4de4-8bf5-4284c24834bc", name => 'Farnham Heath End (0)']` or will `:managed_school_ids` and `:managed_school_names` guaranteed to be the same order?

Reply:
Yes they are guaranteed to be the same order, but I can provide you with a separate array containing hashes with id and name if you prefer